main.py

```diff
@@ -10,6 +11,9 @@
 LOG_FILENAME = 'website.log'
 logging.basicConfig(filename=LOG_FILENAME, level=logging.DEBUG)
 
+STAMP_DIRECTORY = 'output/'
+STAMP_FILE = 'generated_stamp.html'
```

Comment:
This can be title of the webpage.

Reply:
Will be helpful when stampify takes multiple urls as input.

Reply:
Updated accordingly!